builtin/providers/aws/resource_aws_vpc_peering_connection.go

```diff
@@ -98,6 +112,31 @@ func resourceAwsVpcPeeringRead(d *schema.ResourceData, meta interface{}) error {
 	return nil
 }
 
+func resourceVpcPeeringConnectionAccept(conn *ec2.EC2, oldPc *ec2.VpcPeeringConnection, d *schema.ResourceData) error {
+	if oldPc.Status.Code == "pending-acceptance" {
+		log.Printf("[INFO] Accept Vpc Peering Connection with id: %s", d.Id())
+		_, err := conn.AcceptVpcPeeringConnection(d.Id())
+		if err != nil {
+			return fmt.Errorf("Error accepting vpc peering connection: %s", err)
+		}
+
+		pcRaw, _, err := resourceAwsVpcPeeringConnectionStateRefreshFunc(conn, d.Id())()
+		if err != nil {
+			return err
+		}
+		if pcRaw == nil {
+			d.SetId("")
```

Comment:
Will this ever happen? We do the same check in `resourceAwsVpcPeeringRead` before we get to this point.